tests/publiccloud/ssh_interactive_init.pm

```diff
@@ -19,6 +19,14 @@ use utils;
 sub run {
     my ($self, $args) = @_;
 
+    # If someone schedules a publiccloud run with a custom SCHEDULE this causes
+    # the test to break, because we need to pass $args via the main_common.pm.
+    # This check warns about this common pitfall when doing verification runs
+    if (!defined $args) {
+        record_info("Note: Running publiccloud runs with a custom SCHEDULE is not yet supported");
+        die "$args is not defined";
```

Comment:
This will cause in a use of uninitialized string in concatenation error.

Reply:
Fixed